engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,28 +238,59 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float ratio) {
+        return (float) Math.pow(ratio, 0.74f) * 6f + 4f;
+    }
+
+    private float getInteractionRange(float ratio) {
+        return (float) Math.pow(ratio, 0.62f) * 5f;
+    }
+
+    private float getRunFactor(float ratio) {
+        return (float) Math.pow(ratio, 0.68f) * 1.5f;
+    }
+
     @Command(shortDescription = "Sets the height of the player", runOnServer = true,
             requiredPermission = PermissionManager.CHEAT_PERMISSION)
     public String playerHeight(@Sender EntityRef client, @CommandParam("height") float amount) {
-        try {
-            ClientComponent clientComp = client.getComponent(ClientComponent.class);
-            CharacterMovementComponent move = clientComp.character.getComponent(CharacterMovementComponent.class);
-            if (move != null) {
-                float prevHeight = move.height;
-                move.height = amount;
-                clientComp.character.saveComponent(move);
-                LocationComponent loc = client.getComponent(LocationComponent.class);
-                Vector3f currentPosition = loc.getWorldPosition();
-                clientComp.character
-                        .send(new CharacterTeleportEvent(new Vector3f(currentPosition.getX(), currentPosition.getY() + (amount - prevHeight) / 2, currentPosition.getZ())));
-                physics.removeCharacterCollider(clientComp.character);
-                physics.getCharacterCollider(clientComp.character);
-                return "Height of player set to " + amount + " (was " + prevHeight + ")";
+        if (amount >= 1 && amount <= 25) {
+            try {
+                ClientComponent clientComp = client.getComponent(ClientComponent.class);
+                CharacterMovementComponent move = clientComp.character.getComponent(CharacterMovementComponent.class);
+                CharacterComponent charComp = clientComp.character.getComponent(CharacterComponent.class);
+                EntityRef player = clientComp.character;
+                GazeMountPointComponent gazeMountPointComponent = player.getComponent(GazeMountPointComponent.class);
+                Optional<Prefab> prefab = Assets.get(new ResourceUrn("engine:player"), Prefab.class);
+                CharacterMovementComponent moveDefault = prefab.get().getComponent(CharacterMovementComponent.class);
```

Comment:
As we'll need both the default movement component and the default character component later I'll fetch both here:
```suggestion

                Prefab playerDefaults = Assets.getPrefab("engine:player").get();
                CharacterMovementComponent moveDefault = playerDefaults.getComponent(CharacterMovementComponent.class);
                CharacterComponent characterDefault = playerDefaults.getComponent(CharacterComponent.class)
```